### ISSUE TITLE
Check overfitting in mmdet3.3.0 by train curve and validation curve

### ISSUE THREAD

**Issue by Yuga555** [open-mmlab/mmdetection#11560]
To check for model overfitting, I need to plot a curve that shows the training and validation loss or error for each epoch.
But when I run the [mmdetection\tools\train.py](https://github.com/open-mmlab/mmdetection/blob/main/tools/train.py) , log just show train loss but no val loss like this:
```
{"lr": 0.0019819639278557113, "data_time": 0.18926841259002686, "loss": 3.6663841247558593, "loss_rpn_cls": 0.25614564910531046, "loss_rpn_bbox": 0.10180329605937004, "loss_cls": 1.6368599915504456, "acc": 74.853515625, "loss_bbox": 0.8806327283382416, "loss_mask": 0.7909424602985382, "time": 1.5132676649093628, "epoch": 1, "iter": 50, "memory": 4933, "step": 50}
{"lr": 0.0039839679358717435, "data_time": 0.0034419870376586914, "loss": 2.497434725761414, "loss_rpn_cls": 0.08134712882339955, "loss_rpn_bbox": 0.0796768332272768, "loss_cls": 1.0546799910068512, "acc": 75.9765625, "loss_bbox": 0.8015480244159698, "loss_mask": 0.4801827597618103, "time": 1.4791265773773192, "epoch": 1, "iter": 100, "memory": 4933, "step": 100}
{"coco/bbox_mAP": 0.017, "coco/bbox_mAP_50": 0.037, "coco/bbox_mAP_75": 0.013, "coco/bbox_mAP_s": -1.0, "coco/bbox_mAP_m": 0.083, "coco/bbox_mAP_l": 0.017, "coco/segm_mAP": 0.016, "coco/segm_mAP_50": 0.035, "coco/segm_mAP_75": 0.014, "coco/segm_mAP_s": -1.0, "coco/segm_mAP_m": 0.022, "coco/segm_mAP_l": 0.017, "data_time": 0.13413022114680365, "time": 17.261395717278504, "step": 1}
```

I already check these issue [#9650](https://github.com/open-mmlab/mmdetection/issues/9650) [#505](https://github.com/open-mmlab/mmdetection/issues/505) 

In mmdet 2.19.1 document [TUTORIAL 5: CUSTOMIZE RUNTIME SETTINGS - Customize workflow](https://mmdetection.readthedocs.io/en/v2.19.1/tutorials/customize_runtime.html#customize-workflow) show the way to change the workflow setting, but in mmdet dev3.x document [CUSTOMIZE RUNTIME SETTINGS](https://mmdetection.readthedocs.io/en/dev-3.x/advanced_guides/customize_runtime.html#customize-training-schedules), I can't find how to edit workflow setting.

How should I modify mmdet3.3.0 config to obtain validation loss or error for each epoch?

Expressing gratitude in advance for any response!

**Comment by Yuga555** [comment]
Sorry, I didn't notice these

[#9904](https://github.com/open-mmlab/mmdetection/issues/9904)
[#10267](https://github.com/open-mmlab/mmdetection/issues/10267)
